ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -69,13 +69,15 @@ function TokenList({ onTokenClick }: TokenListProps) {
 
     // sort filtered tokens based on the tokenSortConfig in state
```

Comment:
There is a related change from the PR being cherry-picked here: https://github.com/MetaMask/metamask-extension/pull/31302/

See that the `tokenSortConfig.sortCallback === 'alphaNumeric'` block was also removed. I'm guessing we want that removed here as well? It's absent on both v12.15.0 and `main`